packages/sdk-core/src/model/report/BacktraceReport.ts

```diff
@@ -0,0 +1,7 @@
+import { BacktraceAttachment } from './BacktraceAttachment';
+
+export interface BacktraceReport {
+    attributes(): Record<string, unknown>;
+    attachments(): BacktraceAttachment[];
+    toDate(): Record<string, unknown>;
```

Comment:
what is `toDate` exactly? Is there a reason that these are methods, and not `readonly` fields?

Reply:
those are methods and generate something  - toDate convert report to the data object that we can send to Backtrace. Those information will more clean soon

Reply:
Ups, I see - typo